frontend/src/app/modules/modal/modal-header.component.html

```diff
@@ -1,6 +1,7 @@
 <op-icon
+  class="op-modal--header-icon"
```

Comment:
Do I understand correctly that you're not using this class? In that case I wouldn't add it. The rest LGTM

Reply:
Good catch, I first added a class to this but then switched to the align on the parent. I'll remove it